package.json

```diff
@@ -65,7 +65,8 @@
     "metalsmith": "^2.1.0",
     "ora": "^0.2.3",
     "request": "^2.72.0",
-    "root-check": "^1.0.0"
+    "root-check": "^1.0.0",
+    "vorpal": "^1.11.4"
```

Comment:
Yeah, I added it back in so the tests would just execute in whatever format they were left in, but it seemed like they were not relevant anymore. Let me know if I can help with writing the new test suite.